build/BuildDefaults.props

```diff
@@ -3,9 +3,10 @@
     <CLITargets Condition=" '$(CLITargets)' == '' ">Prepare;Compile;Test;Package;Publish</CLITargets>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <IncludeNuGetPackageArchive Condition=" '$(IncludeNuGetPackageArchive)' == '' ">true</IncludeNuGetPackageArchive>
+    <SkipBuildingInstallers Condition=" '$(SkipBuildingInstallers)' == '' AND $(HostRid.StartsWith('rhel.6')) ">true</SkipBuildingInstallers>
```

Comment:
Why use `$(HostRid)` here, but `$(Rid)` below?  Can this be consistent?

Reply:
I am sorry, I've noticed this comment after I've merged it in. It seems that Rid would be more appropriate here. I'll create a new PR for that after I test it.